docs/quickstart/intro.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Redoc quickstart guide
+---
+
+# Redoc quickstart guide
+
+This guide includes step-by-step instructions for how to get started using
+Redoc to render your OpenAPI definition.
+
+Redoc offers multiple options for rendering your OpenAPI definition.
+You should select the option that best fits your needs.
+
+The different options include using the following:
```

Comment:
This can be simplified to just say "The following options are supported".

Reply:
will update